cmd/erasure-zones.go

```diff
@@ -661,7 +661,7 @@ func (z *erasureZones) listObjectsNonSlash(ctx context.Context, bucket, prefix,
 
 	for _, zone := range z.zones {
 		zonesEntryChs = append(zonesEntryChs,
-			zone.startMergeWalksN(ctx, bucket, prefix, "", true, endWalkCh, zone.setDriveCount, false))
+			zone.startMergeWalksN(ctx, bucket, prefix, "", true, endWalkCh, zone.listTolerancePerSet, false))
```

Comment:
Maybe it is better to reduce quorum to be zone.listTolerancePerSet-1

Reply:
No it is already 3 @vadmeste we want 3 - otherwise we would simply list on 2 drives.